Create nft-rfc-008.md

eric-schuh opened the pull request:
Adding first, working draft of nft-rfc-008.

eric-schuh committed:
Create nft-rfc-008.md

First, working draft of nft-rfc-008.

jandrieu commented:
For consistency with the layer model, the title should probably be NFT-RFC-008 NFT Assertions

jandrieu commented on nft-rfc-008.md:
```diff
@@ -0,0 +1,70 @@
+---
+nft-rfc: 8
+title: NFT Resource Verification
```
title: NFT Assertions

ig-shaun replied:
Change title to NFT Assertions

jandrieu commented on nft-rfc-008.md:
```diff
@@ -0,0 +1,70 @@
+modified: 
+---
+
+# NFT RFC 008 NFT Resource Verification
```
# NFT RFC 008 NFT Assertions

ig-shaun reviewed (changes requested):
Needs to be updated with the redraft from Docs.

eric-schuh committed:
Update nft-rfc-008.md

Rev2, fleshed out the Verification Process section, updated language and added an example

eric-schuh committed:
Update nft-rfc-008.md

removed company name and replaced with fictional one

ig-shaun reviewed (approved):
Good first draft
